service/src/main/java/uk/gov/hmcts/reform/fpl/service/JsonOrdersLookupService.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.model.configuration.OrderDefinition;
+
+import java.io.IOException;
+
+import static uk.gov.hmcts.reform.fpl.utils.ResourceReader.readString;
+
+@Service
+public class JsonOrdersLookupService implements OrdersLookupService {
```

Comment:
can we reuse MapperService for this? Possibly add a new method which reads the json file?

Reply:
I would be against this. We will be looking up Json files for Orders more in the future, so it makes sense to be it's own class